src/mongocxx/test/collection_mocked.cpp

```diff
@@ -38,7 +38,7 @@
 #include <mongocxx/read_preference.hpp>
 #include <mongocxx/stdx.hpp>
 #include <mongocxx/options/update.hpp>
-#include <mongocxx/exception/operation_exception.hpp>
+#include <mongocxx/exception/logic_error.hpp>
```

Comment:
For clarity: this is really two changes.  operation_exception was already imported above and thus removed here, and I added logic_error.